test/Faker/Provider/ng_NG/AddressTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Faker\Provider\ng_NG;
+
+use Faker\Generator;
+use Faker\Provider\ng_NG\Address;
+use Faker\Test\Provider\ng_NG\MainTest;
+
+class AddressTest extends MainTest
+{
+
+    /**
+     * @var Generator
+     */
+    private $faker;
+
+    public function setUp()
+    {
+        $faker = new Generator();
+        $faker->addProvider(new Address($faker));
+        $this->faker = $faker;
+    }
+
+    /**
+     *
+     */
+    public function testPostcode()
```

Comment:
You'll need to test at least 2 things:

- `testPostCodeIsntEmpty()`
- `testPostCodeReturnsAValidPostCode()`

Same for the other tests